app/src/main/java/com/duckduckgo/app/browser/errors/ErrorPagePixelFeature.kt

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (c) 2024 DuckDuckGo
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.duckduckgo.app.browser.errors
+
+import com.duckduckgo.anvil.annotations.ContributesRemoteFeature
+import com.duckduckgo.di.scopes.AppScope
+import com.duckduckgo.feature.toggles.api.Toggle
+
+@ContributesRemoteFeature(
+    scope = AppScope::class,
+    featureName = "errorPagePixel",
+)
+interface ErrorPagePixelFeature {
+    @Toggle.DefaultValue(true)
+    fun self(): Toggle
+}
```

Comment:
Why not putting the feature flag under `AndroidBrowserConfigFeature` instead? It's better than having a top level feature.

Reply:
Good idea, I'll change it